types/open-api/.config/schemas.hbs

```diff
@@ -1,3 +1,11 @@
+/** 
+ * This file is automatically generated following the specification in
+ * types/open-api/.config/schemas.hbs and our OpenAPI definitions.
+ *
+ * DO NOT MANUALLY EDIT THIS FILE (nothing will explode, but changes 
+ * will be overwritten).
+ **/
+
```

Comment:
I don't suppose there's a way to leave comments in these `.hbs` files that don't get copied into the generated files, is there? 

I feel like this comment and it's twin in `types.hbs` could use meta-comments like `// THE FOLLOWING COMMENT IS NOT A REAL COMMENT ABOUT THE FILE YOU ARE CURRENTLY READING, IT WILL GET COPIED INTO THE GENERATED FILE.` but that's probably too much to ask.

Reply:
Nope! But it's fine, i feel like whoever reads this either knows what's going on already, or shouldn't be touching this file anyway. I will document how we do type generation as soon as it's stable.